layouts/shortcodes/joinform.html

```
@@ -1,4 +1,4 @@
-<form id="inputForm" action="https://pkic.org/forms" method="POST" class="row g-3 my-2 needs-validation" novalidate>
+<form id="inputForm" action="/api/v1/forms" method="POST" class="row g-3 my-2 needs-validation" novalidate>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)